code/modules/client/verbs/ooc.dm

```diff
@@ -100,9 +97,12 @@ GLOBAL_VAR_INIT(normal_ooc_colour, OOC_COLOR)
 	set name = "Set Your OOC Color"
 	set category = "Preferences"
 
+	/*
+	Not sure how analy retarded you made this exactly TG.
```

Comment:
This was to prevent confusion with admins, iirc. 
Admin ranks have a coloured name, and black/their pref coloured text
OOC Colour is the entire string of ooc text.

Basically, admins have a setting for their OOC name's colour, Byond Members have (had) a setting for the colour of the entire ooc string.

I don't think we want everyone and their mother being able to choose their ooc text colour, so I'd say just lock it behind R_ADMIN, atleast for now.